apps/site/assets/css/_header.scss

```diff
@@ -240,3 +241,96 @@
   font-weight: bold;
   line-height: $headings-line-height;
 }
+
+nav.m-menu, .m-menu__toggle,
+.header-navbar .search {
+  @include media-breakpoint-up(lg) {
+    display: none;
+  }
+}
+.header-navbar {
+  .search {
+    @include media-breakpoint-only(xs) {
+      display: none;
+    }
+  }
+  .header-search__toggle {
+    @include media-breakpoint-up(sm) {
+      display: none;
+    }
+  }
+}
+nav.m-menu {
```

Comment:
The mobile menu is technically still "visible" to assistive tech -- can we hide this in the larger breakpoints? `display: none` or `visibility: hidden` should do it.

<img width="423" alt="image" src="https://user-images.githubusercontent.com/2136286/142255699-e2ec21f0-6242-43d0-af58-de375fb92e78.png">

(screenshot is via the accessibility inspector in Firefox, don't remember what the equivalent in Chrome's dev tools are)